app/Belinska/conftest.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from app.task import Task
+
+from app.to_do_list import ToDoList
+
+
+@pytest.fixture
+def general_task(task_data):
+    task = Task(title=task_data["title"], description=task_data["description"],
+                due_date=task_data["due_date"], priority=task_data["priority"])
+    return task
+
+
+@pytest.fixture
+def specific_task(task_data):
+    task = Task(title=task_data["title"], description=task_data["description"],
+                due_date=task_data["due_date"], priority=task_data["priority"])
+    return task
+
+
+@pytest.fixture
+def empty_todo_list():
+    return ToDoList()
+
+
+@pytest.fixture
+def todo_list_with_tasks():
+    todo_list = ToDoList()
```

Comment:
Here you could use the `empty_todo_list` fixture instead of creating the object again